src/coreclr/src/jit/gentree.h

```diff
@@ -3940,9 +3940,24 @@ struct GenTreeCall final : public GenTree
     // Returns
     //    Type descriptor of the value returned by call
     //
-    // Note:
-    //    Right now implemented only for x64 unix and yet to be
-    //    implemented for other multi-reg target arch (Arm64/Arm32/x86).
+    // TODO-AllArch: enable for all call nodes to unify single-reg and multi-reg returns.
+    const ReturnTypeDesc* GetReturnTypeDesc() const
```

Comment:
Answered in another comment, it also looks like I was not the first one who needed it, pushed another commit that deletes `const_cast` that were added instead of const getter.